lib/jsdom/living/xhr/FormData-impl.js

```diff
@@ -66,6 +66,12 @@ exports.implementation = class FormDataImpl {
       this._entries.push(entry);
     }
   }
+
+  * [Symbol.iterator]() {
+    for (const entry of this._entries) {
+      yield [entry.name, entry.value];
```

Comment:
You might have to manually get the value's wrapper, since webidl2js doesn't try to do that yet. Once that's fixed in webidl2js, we can probably just `return this._entries.entries()` here